specifyweb/frontend/js_src/lib/components/WebLinks/spec.ts

```diff
@@ -161,38 +159,39 @@ function parseField(item: RawWebLink, part: string): ParsedWebLink {
       type: 'FormattedResource',
       formatter: field.title,
     };
-  return {
-    type: 'PromptField',
-    // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-    label: localized(field?.title || field?.name || part),
-  };
+  return { type: 'UrlPart', value: localized('') };
+  // return {
+  //   type: 'PromptField',
+  //   // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+  //   label: localized(field?.title || field?.name || part),
+  // };
 }
 
 function reconstructWeblink(
   parts: RA<ParsedWebLink>
 ): Pick<RawWebLink, 'parameters' | 'url'> {
   const augmented = parts
     .filter((part) => part.type !== 'Field' || part.field.length > 0)
-    .map((parameter, index) =>
+    .map((parameter) =>
       parameter.type === 'Field'
         ? {
             name: parameter.field.map(({ name }) => name).join('.'),
             title: parameter.field.map(({ label }) => label).join(' > '),
             shouldPrompt: false,
```

Comment:
you should remove all references to `shouldPrompt` in this function - since sp7 won't be supporting prompt fields anymore, it should not be modifying the value of `shouldPrompt`, but just leave the way it is in the XML to not break sp6